test/credit_card_sanitizer_test.rb

```diff
@@ -168,6 +168,30 @@ class CreditCardSanitizerTest < MiniTest::Test
         assert_nil @sanitizer.sanitize!("612999921404471347800000")
       end
 
+      describe "exclude html tags" do
+        describe "exclude_html_tags is false" do
+          before do
+            refute @sanitizer.settings[:exclude_html_tags]
+          end
+
+          it "sanitize numbers within html tags" do
+            assert_equal "<div class='m_-=411111▇▇▇▇▇▇1111m_6311419926027290052gmail_signature'>", @sanitizer.sanitize!("<div class='m_-=4111111111111111m_6311419926027290052gmail_signature'>")
+            assert_equal "<p class='411111▇▇▇▇▇▇1111'>", @sanitizer.sanitize!("<p class='4111111111111111'>")
+          end
+        end
+
+        describe "exclude_html_tags is true" do
+          before do
+            @sanitizer = CreditCardSanitizer.new(exclude_html_tags: true)
+          end
+
+          it "sanitize numbers within html tags" do
```

Comment:
I would put in some tests for things that are almost tags but not quite tags, like missing the initial `<` or ending `>` to make sure you're not skipping redaction on them.